Acd 754 (#45)
Actindo-AG/Documentation: DatabaseAndReporting/Integration/05_ManageUserRights.md

[!!Manage the queries](./01_ManageQueries.md)
[!!Manage the query categories](./02_ManageQueryCategories.md)

# Manage the user rights

> The user rights for all modules and plugins installed in the Core1 are managed centrally in the *Settings* module. For detailed information on managing the user rights, see the *Settings* module documentation.

Author: ```suggestion
The user rights for all modules and plugins installed in your Core1 account are managed centrally in the *Settings* module. 
```

    > [Info] It may take some time for the system to display all installed modules and plugins.

    ![Edit group rights](../../Assets/Screenshots/DatabaseAndReporting/EditGroupRights.png "[Edit group rights]")

4. Navigate the folder tree structure on the left side until you locate the *Modules.Actindo.Database and Reporting* folder.
    
5. If necessary, click the rightwards arrow to the left of the *Modules.Actindo.Database and Reporting* folder to display all contained sub-folders.  
    All contained sub-folders are displayed.

    ![Rights DB and reporting](../../Assets/Screenshots/DatabaseAndReporting/EditGroupRightsDatabaseAndReporting.png "[Rights DB and reporting]")

6. Locate the module functions you want to grant access to.

7. Click the rightwards arrow to the left of the corresponding folder (module function) to display all contained sub-folders (access permission types).  
    All contained sub-folders (access permission types) are displayed.

    ![Access permission types](../../Assets/Screenshots/DatabaseAndReporting/AccessPermissionTypes.png "[Access permission types]")

8. Select the checkbox of the applicable folder(s) and/or sub-folders.  

>     The user group, and all users included in it, have been granted access rights to the selected module functions.

Author: ```suggestion
   The user group, and all users included in it, have been granted access rights to the selected module functions.
```
